### PR TITLE
adds logging

### PR DESCRIPTION
**Pull request by arihant-2310**

### CODE REVIEW

**Comment by dauntlessnomad** [comment]
@arihant-2310  do we need to install this additional logger package

**Comment by dauntlessnomad** [comment]
logger is already present in  the common module

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "express-rate-limit": "^5.2.3",
     "helmet": "^4.4.1",
     "husky": "^4.3.8",
+    "logger": "0.0.1",
```
Comment (dauntlessnomad) [inline package.json]: remove this additional package.

**Review by dauntlessnomad — CHANGES_REQUESTED** [review]
remove

**Review by rejahrehim — APPROVED** [review]

**Review by athul — APPROVED** [review]

**Review by dauntlessnomad — APPROVED** [review]

**Pull request merged**